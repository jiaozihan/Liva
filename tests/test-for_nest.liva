class test {
  void main(){
	int i;
	int j;
	for(i = 0; i < 10; i = i + 1)
		for(j = 11; j < 13; j = j + 1)
			print("i = ", i, " ", "j = ", j, "\n");
  }	
}