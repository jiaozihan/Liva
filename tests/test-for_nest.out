i = 0 j = 11
i = 0 j = 12
i = 1 j = 11
i = 1 j = 12
i = 2 j = 11
i = 2 j = 12
i = 3 j = 11
i = 3 j = 12
i = 4 j = 11
i = 4 j = 12
i = 5 j = 11
i = 5 j = 12
i = 6 j = 11
i = 6 j = 12
i = 7 j = 11
i = 7 j = 12
i = 8 j = 11
i = 8 j = 12
i = 9 j = 11
i = 9 j = 12